### PR TITLE
Edits to Pathogensis, especially -omics sections

### LINKED ISSUE #612: Intro / Pathogenesis Section Structures
**Is your feature request related to a problem? Please describe.**

Right now, the Introduction and Pathogenesis sections have a bit of duplication (and in areas where they don't, there are gaps in the pathogenesis section).

**Describe the solution you'd like**

We should restructure things to cover things more or less at one place. It strikes me that pulling sections 1.2 and 1.3 from the intro and placing them into the pathogenesis section (merging with what's already there) might be the most appropriate solution. It means that - of the areas discussed in the intro - basic biomedicine and public health would get their section together (pathogenesis), and then the biotechnology focus (which is, from what I can tell, our main focus right now) would get two sections (one each for diagnostics and treatments).

**Describe alternatives you've considered**

We could eliminate pathogenesis and deal with everything in this section in the intro. We could alternatively have a pathogenesis section focused on basic biomedicine and another section focused on the public health side of things. If we want to do this, we'd want to make clear that we are not aiming to be comprehensive in either of those.

**Additional context**

I have some other thoughts as we do this:
* I think the systems-level approaches section should also be better integrated into the biomedical portions around infection, viral genome, etc. It's not clear to me that it makes a lot of sense as a separate section given the current structure.
* I think the PPI and transcriptomics sections would be better integrated into a section on viral-host interactions & response.
* I'm not sure we need Figure 2B at this location - it gets very deep into data before we have some of the introductory figures that were made ( https://github.com/greenelab/covid19-review/tree/master/content/images ). It's possible that my opinion would change with the described restructuring though.

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This PR is mostly just a text clean-up for pathogenesis. However, I did receive feedback that the -omics sections were a bit more detail-oriented relative to the rest of the manuscript, and didn't swing out to the "big picture" approach we've taken most places. I tried to make these edits but am definitely open to feedback about whether this was successful (e.g., it seems like maybe we should just leave MOI in? I am super on the fence about swapping it for "level of exposure").

### Related issues

closes #612 

### Suggested reviewers (optional)

Original authors: @rays1987 @bansalvi @marouenbg -- This is definitely not a decision that this stuff should be cut! Just a suggestion. Very happy to hear what you think needs to stay. I actually think the analysis and figure are very interesting, but unfortunately not sure they work with the scope of the manuscript as it stands right now. I feel like you could make a separate blog post/paper about this as more data comes out!!!
@cgreene 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by cgreene — APPROVED** [review]
🚀  Looks good!

File: content/07.pathogenesis.md [hunk 1]
```diff
@@ -197,87 +196,83 @@ Omics-based studies have also provided meaningful information regarding host imm
...
-All infections were performed at a high MOI (2–5), with the exception of A549-ACE2 cells, which were infected with SARS-CoV-2 at both low (0.2) and high (2) MOI.
-Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed at 24 hours post-infection (hpi) of all the viruses except for IAV, which was formed at 9-12 hours.
 The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes from an animal model (ferret), and lung samples from two COVID-19 patients.
-Additionally, they measured transcriptional responses to IAV in the NHBE cells and the nasal washes of ferrets.
 Differential expression (DE) analysis was then carried out to compare infected cells with control cells that underwent only a mock treatment.
 n the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
```
Comment (bansalvi) [inline content/07.pathogenesis.md:211]: I think there is an **I** missing.

Comment (rando2) [reply content/07.pathogenesis.md:211]:
```suggestion
In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
```
Thanks @bansalvi!

File: content/07.pathogenesis.md [hunk 2]
```diff
@@ -148,16 +147,16 @@ Cytokines have been investigated for their role in the immune response to lung i
 Dysregulation of the inflammatory response, including elevated levels of pro-inflammatory cytokines, is found in patients with Acute Respiratory Distress Syndrome (ARDS), which is a severe condition that can arise from pneumonia, SARS, and COVID-19 [@doi:10.1159/000087104].
```
Comment (RLordan) [inline content/07.pathogenesis.md:147]: I think you abbreviate ARDS in intro section

File: content/07.pathogenesis.md [hunk 3]
```diff
@@ -148,16 +147,16 @@ Cytokines have been investigated for their role in the immune response to lung i
...
 IL-6 and TNF are pro-inflammatory cytokines, while IL-1&beta; is anti-inflammatory [@doi:10.1001/archinte.167.15.1655].
 However, this study reported that unbalanced pro-/anti-inflammatory cytokine profiles were rare, although they measured only the three cytokines listed above.
 Prior work therefore made it clear that pulmonary infection and injury were associated with systemic inflammation through sepsis.
-While IL-6 is a biomarker sometimes used to assess cytokine storm activity in sepsis [@doi:10.1128/MMBR.05015-11], the relationship between cytokine profiles and the risks associated with sepsis may be more complex. 
+While IL-6 is a biomarker sometimes used to assess cytokine storm activity in sepsis [@doi:10.1128/MMBR.05015-11], the relationship between cytokine profiles and the risks associated with sepsis may be more complex.
 IL-6 is a pleiotropic cytokine that plays a role in both the inflammatory and anti-inflammatory responses and is associated with both healthy and pathological responses to viral threat [@doi:10.3389/fmicb.2019.01057].
```
Comment (RLordan) [inline content/07.pathogenesis.md:160]:
```suggestion
IL-6 is a pleiotropic cytokine that plays an integral role in both the inflammatory and anti-inflammatory responses and is associated with both healthy and pathological responses to viral threat [@doi:10.3389/fmicb.2019.01057].
```

File: content/07.pathogenesis.md [hunk 4]
```diff
@@ -148,16 +147,16 @@ Cytokines have been investigated for their role in the immune response to lung i
...
 In this way, interleukins like IL-6 are integral to both the pro- and anti-inflammatory responses.
```
Comment (RLordan) [inline content/07.pathogenesis.md:161]: Maybe delete this sentence as per change above?

**Review by RLordan — COMMENTED** [review]
Nothing major, small edits.

File: content/07.pathogenesis.md (hunk 2 above)
Comment (rando2) [reply content/07.pathogenesis.md:147]:
```suggestion
Dysregulation of the inflammatory response, including elevated levels of pro-inflammatory cytokines, is found in patients with ARDS, which is a severe condition that can arise from pneumonia, SARS, and COVID-19 [@doi:10.1159/000087104].
```

**Comment by rando2** [comment]
I'm going to go ahead and merge this, but please feel free to open additional PRs if there is material you'd like to change!

**Pull request merged**